client/ayon_unreal/addon.py

```diff
@@ -14,6 +14,8 @@ class UnrealAddon(AYONAddon, IHostAddon):
 
     def initialize(self, settings):
         self.enabled = settings[self.name]['enabled']
+        if 'UE_PYTHONPATH' not in os.environ:
+            os.environ['UE_PYTHONPATH'] = os.environ['PYTHONPATH']
```

Comment:
I don't think this is going to do much for you withing the unreal addon, and since slate will be running without this addon. This will need to be done within slate. 

See `add_implementation_envs` below. You'll see the the UE_PYTHONPATH is being set there.

`add_implementation_envs` is a special method for setting the environment but it's only for host addons.